integration/pom.xml

```diff
@@ -24,6 +24,7 @@
         <wildflyTargetDir>${basedir}/target/wildfly-${wildfly.version}</wildflyTargetDir>
         <wildflyModulesDir>${wildflyTargetDir}/modules/system/layers/base</wildflyModulesDir>
         <hibernate-validator-parent.path>..</hibernate-validator-parent.path>
+        <automatic.module.name>org.hibernate.validator.integration</automatic.module.name>
```

Comment:
I don't think this one is necessary: we don't publish anything and it's not supposed to be used.